moveit_ros/planning/planning_pipeline/src/planning_pipeline.cpp

```diff
@@ -110,42 +110,48 @@ void planning_pipeline::PlanningPipeline::configure()
   catch (pluginlib::PluginlibException& ex)
   {
     RCLCPP_FATAL(LOGGER, "Exception while creating planning plugin loader %s", ex.what());
+    throw;
   }
 
   std::vector<std::string> classes;
   if (planner_plugin_loader_)
     classes = planner_plugin_loader_->getDeclaredClasses();
-  if (planner_plugin_name_.empty() && classes.size() == 1)
-  {
-    planner_plugin_name_ = classes[0];
-    RCLCPP_INFO(
-        LOGGER,
-        "No '~planning_plugin' parameter specified, but only '%s' planning plugin is available. Using that one.",
-        planner_plugin_name_.c_str());
-  }
-  if (planner_plugin_name_.empty() && classes.size() > 1)
+
+  if (planner_plugin_name_.empty())
```

Comment:
Alternatively we could consider defaulting to OMPL with `planner_plugin_name_` :thinking: